spec/01-lexical-syntax.md

```diff
@@ -83,7 +83,7 @@ The rules for pattern matching further distinguish between
 _variable identifiers_, which start with a lower case letter
 or `_`, and _constant identifiers_, which do not.
 
-For this purpose, lower case letter don't only include a-z,
+For this purpose, lower case letter doesn't only include a-z,
```

Comment:
I think it means the other way around: `"lower[-]case letter[s]" don't only include` referring to the prenominate category.

The ill hyphenated "lower case" is worth fixing.

Reply:
To-day, I would go with `lowercase`. merriam-webster says 1683, not an ill year.

Reply:
"illy hyphenated"